keycloak/keycloak-operator
# INTLY-3699 fix prometheus expressions and update grafana dashboard

## JIRA ID

INTLY-3699

## Additional Information

This fixes the Prometheus rules so that they can be imported. Also fixes a problem where the Grafana dashboard caused constant reconciliation (because both operators, keycloak and grafana modified it). I've also updated to a better dashboard.

## Verification Steps

1. Deploy the application monitoring stack
2. Deploy keycloak (or rhsso) in another namespace (it must have the monitoring-key=middleware label)
3. Make sure the dashboard and prometheus rules are imported
4. There will be no data to display because that would require the metrics SPI
